.travis.yml

```diff
@@ -4,14 +4,16 @@ rvm:
   - 2.4
   - 2.5
   - 2.6
+  - 2.7
```

Comment:
You may want to add an `allow_failures` sections for Ruby 2.7, since it is still in development.